src/main/resources/config.yml

```diff
@@ -509,16 +509,35 @@ custom-events:
   #Reward a player a price in your economy currency every time he is still alive after the interval (in seconds)
   time:
     enable: false
+    # Do you want to reward the player with a currency ?
+    isEconomy: false
+    # If above is true, they will receive the amount bellow
     reward: 100
+    # if isEconomy is false, enter a list of commands that should be executed instead (the / before is ignored)
+    commands:
+      - msg %name% Congratulation you survived !
+      - /give %name% gold_ingot 1
     interval: 600
+
   #Reward a player a price in your economy currency every time he kills an enemy
   kill:
     enable: false
+    # Do you want to reward the player with a currency ?
+    isEconomy: false
+    # If above is true, they will receive the amount bellow
     reward: 500
+    # if isEconomy is false, enter a list of commands that should be executed instead (the / before is ignored)
+    commands:
+      - msg %name% You got a kill !
+      - /give %name% diamond 5
   #Reward a price in your economy currency to the winners of the game
   win:
     enable: false
+    isEconomy: false
     reward: 1000
+    commands:
+      - msg %name% You won the game !
+      - /give %name% diamond 128
```

Comment:
Instead of adding a `isEconomy` option maybe execute both? And when reward is 0 or command list is empty that part doesn't get triggered.